editor/test/reducer.js

```diff
@@ -815,7 +815,7 @@ describe( 'state', () => {
 		it( 'should apply all defaults', () => {
 			const state = preferences( undefined, {} );
 
-			expect( state ).toEqual( { blockUsage: {}, recentlyUsedBlocks: [], mode: 'visual', isSidebarOpened: true, panels: { 'post-status': true } } );
+			expect( state ).toEqual( { blockUsage: {}, recentlyUsedBlocks: [], mode: 'visual', isSidebarOpened: true, isExtendedSettingsOpened: true, panels: { 'post-status': true } } );
 		} );
```

Comment:
Can we add 2 tests that verify changes applied to the reducer when `TOGGLE_EXTENDED_SETTINGS` action is triggered? It would be very similar to the existing tests for `TOGGLE_SIDEBAR_PANEL` action.